msteams-platform/feedback.md

```diff
@@ -1,29 +1,36 @@
-|       **Documentation updates**        | Select the Edit link on an article and submit a pull request to the [Microsoft Teams documentation GitHub](https://github.com/MicrosoftDocs/msteams-docs) repo.                                                                                                                                                                      |
-|       **Community chat group**        | Select **SIGN IN TO START TALKING** to join the informal community chat room for [Microsoft Teams app developers](https://gitter.im/OfficeDev/MicrosoftTeamsAppDev), if you are logging in for the first time. Select **JOIN ROOM**, if you are already logged in. |
-|          **General questions**         |          General questions about the Microsoft Teams developer platform, please send an email to [Microsoft Teams Community Help](mailto:microsoftteamsdev@microsoft.com). We encourage posting questions on channels mentioned above and use email only if none of the above are applicable.                                                                                                                                                                          |
-
-## Product Support channels
+|         **Community support**          |<ol><li> Use the `botframework & microsoft-teams` tag to post how-to questions about the SDK at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams). Stack Overflow has guidelines, such as requiring a descriptive title, a complete and concise problem statement, and sufficient details to reproduce your issue. Feature requests or overly broad questions are off-topic. For guidance on how to ask your queries, users must visit the [Stack Overflow Help Center](https://stackoverflow.com/help/how-to-ask). </li>                                                                                                                                                                       <li>  Questions can also be posted at [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) using the `office-teams-app-dev` tag. We love Stack Overflow and our extensive support is assured for customers who ask questions there. However, Stack Overflow has specific criteria about what questions are appropriate for the community and [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) has an open policy regarding this.  </li> </ol>                                                                                                  |
```

Comment:
Please rephrase as follows:

You can post questions on one of the following:

- [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams): Use the microsoft-teams tag to post the questions.
   > [!NOTE]
   > Ensure that you follow the Stack Overflow guidelines such as provide a descriptive title, a complete and concise problem statement, and sufficient detail to reproduce your issue. Feature requests or overly broad questions are off-topic. if you are a new user, for more information, see [Stack Overflow Help Center](https://stackoverflow.com/help/how-to-ask).
   
- Rephrase in the similar way for point 2.